app/lib/tool/test_profile/import_source.dart

```diff
@@ -65,8 +65,10 @@ class _PubDevImportSource implements ImportSource {
     final file = File(p.join(archiveCachePath, archiveName));
     // download package archive if not already in the cache
     if (!await file.exists()) {
+      // TODO: Use archive_url from version-listing, that is stable!
```

Comment:
Nit: can't we use the same method to generate it?

Reply:
Which method? It didn't work here because I changed it :rofl: and pub.dev doesn't support the new end-point yet.

Reply:
Ah, got it.